docker-compose.dev.yml

```diff
@@ -22,6 +22,9 @@ services:
       - './broker-cli:/home/app/broker-cli/'
       - './proto:/home/app/proto/'
       - './scripts:/home/app/scripts/'
+    environment:
```

Comment:
Moved this environment variable because it is only valid if we are using auth for a local relayer